src/main/java/com/dnd/modutime/util/RealTimeProvider.java

```diff
@@ -1,12 +1,16 @@
 package com.dnd.modutime.util;
 
 import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+
 import org.springframework.stereotype.Component;
 
 @Component
 public class RealTimeProvider implements TimeProvider {
     @Override
     public LocalDateTime getCurrentLocalDateTime() {
-        return LocalDateTime.now();
+        ZoneOffset koreaZoneOffset = ZoneOffset.of("+09:00");
+        return ZonedDateTime.now(koreaZoneOffset).toLocalDateTime();
```

Comment:
r: 시차를 적용하는 것보단 타임존을 설정해주는 것이 어떨까요?
https://www.daleseo.com/java8-zoned-date-time/

```suggestion
        ZoneId seoulZoneId = ZoneId.of("Asia/Seoul");
        return ZonedDateTime.now(seoulZoneId).toLocalDateTime();
```